letsencrypt/DOCS.md

```diff
@@ -23,26 +29,39 @@ To use this add-on, you have two options on how to get your certificate:
 - Allows to request wildcard certificates (*.yourdomain.com)
 - Doesn’t need you to open a port to your Home Assistant host on your router.
 
-### You always need to provide the following entries within the configuration
+### DNS providers
 
-```yaml
-email: your@email.com
-domains:
-  # use "*.yourdomain.com" for wildcard certificates.
-  - yourdomain.com
-challenge: http OR dns
-```
-
-IF you choose `dns` as `challenge`, you will also need to fill:
+<details>
+  <summary>Supported DNS providerss</summary>
 
-```yaml
-# Add the dnsprovider of your choice from the list of "Supported DNS providers" below
-dns:
-  provider: ""
+```txt
+dns-azure
+dns-cloudflare
+dns-cloudxns
+dns-digitalocean
+dns-directadmin
+dns-dnsimple
+dns-dnsmadeeasy
+dns-gehirn
+dns-google
+dns-hetzner
+dns-linode
+dns-luadns
+dns-njalla
+dns-nsone
+dns-ovh
+dns-rfc2136
+dns-route53
+dns-sakuracloud
+dns-netcup
+dns-gandi
+dns-transip
+dns-inwx
```

Comment:
Change it to the (so far) up-to-date list:
```suggestion
dns-azure
dns-cloudflare
dns-desec
dns-digitalocean
dns-directadmin
dns-dnsimple
dns-dnsmadeeasy
dns-duckdns
dns-dreamhost
dns-gehirn
dns-google
dns-hetzner
dns-infomaniak
dns-linode
dns-luadns
dns-njalla
dns-nsone
dns-ovh
dns-rfc2136
dns-route53
dns-sakuracloud
dns-namecheap
dns-netcup
dns-gandi
dns-transip
dns-inwx
dns-porkbun
```

Reply:
Done.